### PR TITLE
Update feature stability for 1.2

### PR DESCRIPTION
**Pull request by duderino**
Fix https://github.com/istio/istio.io/issues/4423

### CODE REVIEW

File: content/about/feature-stages/index.md [hunk 1]
```diff
@@ -90,7 +94,6 @@ Below is our list of existing features and their current phases. This informatio
 | [Kubernetes: Envoy Installation and Traffic Interception](/docs/setup/kubernetes/)        | Stable
 | [Kubernetes: Istio Control Plane Installation](/docs/setup/kubernetes/) | Stable
 | [Attribute Expression Language](/docs/reference/config/policy-and-telemetry/expression-language/)        | Stable
-| [Mixer In-Process Adapter Authoring Model](/blog/2017/adapter-model/)        | Deprecated
```
Comment (sdake) [inline content/about/feature-stages/index.md]: instead of saying Deprecated, would it make sense to state `Deprecated and removed` so operators know a deprecated feature is no longer available?  In 1.3, we could move the line entirely.

File: content/about/feature-stages/index.md [hunk 2]
```diff
@@ -57,11 +60,12 @@ Below is our list of existing features and their current phases. This informatio
 | [Client and Server Telemetry Reporting](/docs/concepts/policies-and-telemetry/) | Stable
 | [Service Dashboard in Grafana](/docs/tasks/telemetry/metrics/using-istio-dashboard/) | Stable
 | [Istio Component Dashboard in Grafana](/docs/tasks/telemetry/metrics/using-istio-dashboard/) | Stable
+| [Distributed Tracing to Zipkin / Jaeger](/docs/tasks/telemetry/distributed-tracing/) | Stable
```
Comment (sdake) [inline content/about/feature-stages/index.md]: Zipkin causes a lot of confusion in the telemetry issues.  I think this would be better worded as `Distributed tracing using the ZipKin API to Jaeger`

thoughts @douglas-reid ?

File: content/about/feature-stages/index.md (hunk 1 above)
Comment (duderino) [reply content/about/feature-stages/index.md]: Happy to comply.  @douglas-reid thoughts?

Comment (duderino) [reply content/about/feature-stages/index.md]: Happy to comply, @douglas-reid thoughts?

File: content/about/feature-stages/index.md (hunk 2 above)
Comment (douglas-reid) [reply content/about/feature-stages/index.md]: The difference between this line and the one below it are hard to parse out. I'd suggest collapsing and just calling it Distributed Tracing.

File: content/about/feature-stages/index.md [hunk 3]
```diff
@@ -57,11 +60,12 @@ Below is our list of existing features and their current phases. This informatio
...
+| [Service Tracing](/docs/tasks/telemetry/distributed-tracing/) | Stable
 | [Stackdriver Integration](/docs/reference/config/policy-and-telemetry/adapters/stackdriver/) | Beta
-| [Distributed Tracing to Zipkin / Jaeger](/docs/tasks/telemetry/distributed-tracing/) | Beta
-| [Service Tracing](/docs/tasks/telemetry/distributed-tracing/) | Beta
 | [Logging with Fluentd](/docs/tasks/telemetry/logs/fluentd/) | Beta
 | [Trace Sampling](/docs/tasks/telemetry/distributed-tracing/overview/#trace-sampling) | Beta
+| Prometheus Operator | Alpha
```
Comment (douglas-reid) [inline content/about/feature-stages/index.md]: I'd maybe call this `Prometheus Operator Support`.

File: content/about/feature-stages/index.md (hunk 1 above)
Comment (douglas-reid) [reply content/about/feature-stages/index.md]: I don't think we need to keep it. It was `Deprecated` one release ago. No new in-process adapters have been accepted. And we've been actively rejecting them for (most of) 2 release cycles. I think it is time to just remove it.

File: content/about/feature-stages/index.md [hunk 4]
```diff
@@ -74,12 +78,10 @@ Below is our list of existing features and their current phases. This informatio
 | [Kubernetes: Service Credential Distribution](/docs/concepts/security/#pki)   | Stable
 | [VM: Service Credential Distribution](/docs/concepts/security/#pki)         | Beta
 | [Mutual TLS Migration](/docs/tasks/security/mtls-migration)    | Beta
-| [Authentication policy](/docs/concepts/security/#authentication-policies)  | Alpha
+| [Authorization (RBAC)](/docs/concepts/security/#authorization)   | Beta
```
Comment (linsun) [inline content/about/feature-stages/index.md]: do we want to call this beta?  I feel the new API is under reviewed last week.

File: content/about/feature-stages/index.md (hunk 1 above)
Comment (linsun) [reply content/about/feature-stages/index.md]: agreed, we should just remove it

File: content/about/feature-stages/index.md (hunk 3 above)
Comment (linsun) [reply content/about/feature-stages/index.md]: is this new?  I don't recall a doc page for it, neither in my k8s cluster when istio default profile is installed @douglas-reid

Comment (linsun) [reply content/about/feature-stages/index.md]: @howardjohn reminded me this is for new installer, so i think we should not mention this for 1.2

File: content/about/feature-stages/index.md [hunk 5]
```diff
@@ -74,23 +77,22 @@ Below is our list of existing features and their current phases. This informatio
...
+| [Authorization (RBAC)](/docs/concepts/security/#authorization)   | Alpha
 | [End User (JWT) Authentication](/docs/concepts/security/#authentication)  | Alpha
 | [OPA Checker](/docs/reference/config/policy-and-telemetry/adapters/opa/)    | Alpha
-| [Authorization (RBAC)](/docs/concepts/security/#authorization)   | Alpha
-| [TCP Authorization (RBAC)](/docs/tasks/security/authz-tcp) | Alpha
-| [Cert management on Ingress Gateway](/docs/tasks/traffic-management/ingress/secure-ingress-sds) | Alpha
 | [Vault integration](/docs/tasks/security/vault-ca) | Alpha
 | [SDS Integration](/docs/tasks/security/auth-sds/) | Alpha
 
+The 'Authorization (RBAC)' runtime is considered Beta, but it is advertised as Alpha since its API is still subject to backward incompatible change.
```
Comment (duderino) [inline content/about/feature-stages/index.md]: @liminw @linsun I'm proposing this as a way of indicating that the impl is more stable, but the API is still subject to change

Comment (liminw) [reply content/about/feature-stages/index.md]: Sounds good to me. Thanks @duderino

File: content/about/feature-stages/index.md (hunk 1 above)
Comment (duderino) [reply content/about/feature-stages/index.md]: removed

File: content/about/feature-stages/index.md (hunk 2 above)
Comment (duderino) [reply content/about/feature-stages/index.md]: Changed to just Distributed Tracing

File: content/about/feature-stages/index.md (hunk 3 above)
Comment (duderino) [reply content/about/feature-stages/index.md]: Removed mention of the Prometheus Operator

File: content/about/feature-stages/index.md (hunk 4 above)
Comment (duderino) [reply content/about/feature-stages/index.md]: Set it back to Alpha and added a note that the impl could be considered Beta but the API is still subject to change like an Alpha

File: content/about/feature-stages/index.md (hunk 5 above)
Comment (rcaballeromx) [reply content/about/feature-stages/index.md]: @duderino Maybe this wording would work better:
```suggestion
The 'Authorization (RBAC)' runtime is considered Beta. However, its API is still subject to a backwards incompatible change.
Due to this, we advertise it as Alpha.
```

File: content/about/feature-stages/index.md [hunk 6]
```diff
@@ -42,10 +42,13 @@ Below is our list of existing features and their current phases. This informatio
...
 | TLS termination and SNI Support in Gateways | Stable
+| SNI (multiple certs) at ingress | Stable
+| [Locality load balancing](/docs/ops/traffic-management/locality-load-balancing/) | Beta
 | Enabling custom filters in Envoy | Alpha
 | CNI container interface | Alpha
-| Sidecar API | Alpha
-| SNI (multiple certs) at ingress | Alpha
+| [Sidecar API](/docs/reference/config/networking/v1alpha3/sidecar/) | Alpha
+
+The `Sidecar` runtime is considered Beta, but it is advertised as Alpha since its API is still subject to backward incompatible change.
```
Comment (rcaballeromx) [inline content/about/feature-stages/index.md]:
```suggestion
The `Sidecar` runtime is considered Beta. However, its API is still subject to a backwards incompatible change.
Due to this, we advertise it as Alpha.
```

Comment (duderino) [reply content/about/feature-stages/index.md]: done!
